app/src/main/java/net/gsantner/opoc/util/ShareUtil.java

```diff
@@ -1165,4 +1168,26 @@ public void callTelephoneNumber(String telNo, boolean... directCall) {
             activity.startActivity(intent);
         }
     }
+
+    public StringBuilder fetchContentOfFile(File file){
```

Comment:
Use FileUtils::readTextfileFast, so that function is basically already available

Reply:
Fix in wladokacur/markor@6c6f39893eb4315466faf99062318f8da95c45a8